conda-forge.yml

```diff
@@ -1 +1,5 @@
+build_platform:
```

Comment:
Does this mean that it doesn't build on things that aren't linux?

Reply:
As far as I know, it builds on osx_64 too. We could add osx_arm64 in another PR.

Reply:
OK. I just wasn't sure whether `build_platform` was an inclusive list or an exclusive list.

Reply:
It looks like osx and win builds are queued, so that's evidently OK.